assets/components/Navbar.js

```diff
@@ -1,10 +1,14 @@
-const Navbar = () => {
-	return `
-  <header class="w-100">
-    <nav class=" navbar navbar-expand-lg navbar-light bg-light">
+import SearchBar from "./SearchBar.js";
+
+const Navbar = (params) => {
+  const { isSearchable } = params;
+  return `
+  <header class="position-fixed top-0 start-0 w-100" style="z-index:999">
+    <nav class="z-index-master navbar navbar-expand-lg navbar-light bg-light">
       <a class="navbar-brand fs-1 fw-bold mx-4" href="#">Outsafe BC</a>
       <div class="input-group">
     </nav>
+    ${ isSearchable ? SearchBar(params): "" }
```

Comment:
Let's move the SearchBar that we want in the Home (View Hazards) separated from  the Navbar, the Navbar will be shared on all the pages and we don't need this specific functionality in generic components.